### ISSUE TITLE
using glm with mlr?

### ISSUE THREAD

**Issue by andreashandel** [mlr-org/mlr#837]
Hi, it's me again :)

My question this time: I have some outcome data that decently follows a gamma distribution, so I want to fit a GLM with a gamma distribution. I would like to do it in the mlr framework to be able to use all the 'goodies' that come with mlr (specifically in this case automated feature selection). Unfortunately glm is not part of the learners. I'm surprised to not see it on the list. 

I searched around in the 'issues' posts a bit and found some related discussion about poisson distributed data and a 'hack' for glmnet. Is there a way I could get mlr to 'talk to' glm? Maybe somehow by defining a performance measure that would correspond to the gamma link function? Or some other way? Thanks!

**Comment by larskotthoff** [comment]
There's `(classif|regr).glmboost` and `(classif|regr).glmnet`.

**Comment by andreashandel** [comment]
Yes but they don't allow gamma distributed outcomes, as far as I'm aware?

**Comment by berndbischl** [comment]
I guess we could add it. We do have regr.lm, which of course offers less flexibility.

Andreas, why not add this locally as a custom learner as explained in the tutorial. 
Then do a PR. And I can have someone iterate over it to improve it

**Comment by andreashandel** [comment]
@berndbischl while I'm in principle all in favor of your suggestion that i build a custom learner, the combination of my available time and my coding prowess suggest that it's unlikely this could happen anytime soon.

**Comment by berndbischl** [comment]
ok, we will take this over then. but it can take a couple of days as there is other stuff on the prio list.

the main arg of glm seems to the the family list? what kind of options do you think are reasonable to inlcude?

**Comment by andreashandel** [comment]
@berndbischl That sounds great, and I'm happy to wait until it's available, no problem if it takes a while. The main 'functionality' I want to get from glm is having access to different families/link-functions. When I use glm() by itself, I almost never change any other possible settings and just use whatever defaults glm() sets. So all those could be (for my purpose anyway) be set by mlr to the glm() defaults.

Also, based on @larskotthoff comment above, I took a look at the (so far unfamiliar to me) regr.glmboost and the mboost package. I saw in the help file that glmboost/mboost also has a 'family' setting, with what seems support for more families/link functions than glm(). I tried setting different families within mlr like so:
learner_name = "regr.glmboost"
mylearner = makeLearner(learner_name, family = Laplace() )
But that didn't seem to work. I don't need it to work, just figured I let you know what I tried.
